json-guides/retry-timeout.json

```diff
@@ -91,6 +91,64 @@
                 }
             ]
         },
+        {
+            "name": "TimeoutAnnotation",
+            "title": "Adding the @Timeout annotation",
+            "description": [
+                "Now that you've seen the application easily fail when it loads for an indefinite amount of time, let's apply a Timeout policy to to limit the time the request is made to a transaction history and to prevent the page from loading forever.",
+                "With a Timeout policy, the microservice fail with <code>TimeoutException</code> after the set amount of time has passed when the request to a transaction history is made.",
```

Comment:
With a Timeout policy, the microservice *fails* with *a* `TimeoutException` *when the set amount of time passes after the request is made*